keel-veto/src/test/kotlin/com/netflix/spinnaker/keel/veto/ApplicationVetoTests.kt

```diff
@@ -115,53 +83,5 @@ class ApplicationVetoTests : JUnit5Minutests {
           .hasSize(0)
       }
     }
-
```

Comment:
These weren't needed because I made sure we always pass the app name into the veto.